.github/ISSUE_TEMPLATE/block-snap.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Block a Snap
+about: Required for blocking an existing Snap (still listed in the registry but not accessible)
+title: "[Block Snap] Name of Snap here"
+labels: allowlist,allowlist-block-snap
+assignees: ''
+
+---
+
+All items in the list below needs to be satisifed, with 1 approval from someone on the MetaMask Snaps team.
```

Comment:
Maybe something like this makes more sense (here and in the other templates)? Up to you though!

```suggestion
## Description

...

## Checklist

<!--
All items in the list below needs to be satisfied.
-->
```

I think the "with 1 approval from someone on the MetaMask Snaps team" is redundant, since all pull requests require a review regardless.